openedx/features/redhouse_features/registration/models.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.db import models
+
+
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+USER_TYPES = [
+        (b'student', b'Student'),
+        (b'staff', b'Staff')
+    ]
+ORGANIZATIONS_TYPES = [
+        (b'corporate', b'Corporate'),
+        (b'school district', b'School District'),
+        (b'private school', b'Private School'),
+        (b'higher ed', b'Higher ED'),
+        (b'training', b'Training')
+    ]
+
+
+class AdditionalRegistrationFields(models.Model):
+    """
+        This model contains two extra fields that will be saved when a user registers.
+        The form that wraps this model is in the forms.py file.
+        """
+    user = models.OneToOneField(USER_MODEL, null=True, on_delete=models.SET_NULL)
```

Comment:
I believe we need models.CASCADE here as we don't want to store data of deleted accounts.